### DISCUSSION TITLE
Rate & Give your valuable feedback ✨

### DISCUSSION THREAD

**Discussion by pranjal-barnwal** [pranjal-barnwal/minimal-kiwi#2] [Q&A]
<h1 align="center">
  <br>
    <img src="https://github.com/pranjal-barnwal/pranjal-barnwal/assets/71400605/60cbe9e9-57e0-45dd-811d-36cf0e5d4435" alt="logo" width="200">
  <br><br>
  <a href="https://marketplace.visualstudio.com/items?itemName=PranjalKumar.minimal-kiwi">
  Minimal Kiwi | VS-Code Theme
  </a>
  <br>
<img src="https://github.com/pranjal-barnwal/pranjal-barnwal/assets/71400605/bc5849d4-74c9-4bec-bedf-69576141cc37" alt="Minimal Kiwi Cover by Pranjal Kumar (@pranjal-barnwal)">

</h1>

**Comment by Black0S** [comment]
I really love this theme’s colors. I’m sharing with you a quick preview of how I use it.  
Thank you very much; the color palette is truly amazing.  
I can’t wait to see what comes next and discover your future creations.

![CleanShot 2025-12-11 at 10 31 19](https://github.com/user-attachments/assets/0bbafac6-7df9-4f1a-a478-f94c7df40135)
![CleanShot 2025-12-11 at 10 32 43](https://github.com/user-attachments/assets/115597a0-c35d-4f94-b57f-d736b925043b)
![CleanShot 2025-12-11 at 10 41 00](https://github.com/user-attachments/assets/1fa5fd6f-c604-4d92-9488-e8193a87874d)
